src/applications/find-forms/components/FindVaForms.jsx

```diff
@@ -13,111 +19,147 @@ const onFeaturedContentClick = header => () => {
   });
 };
 
-export default () => (
-  <>
-    <SearchForm />
-    <SearchResults />
-    <h2>Frequently used VA forms</h2>
-    <p>
-      You can now do many form-based tasks online, like filing a disability
-      claim and applying for the GI Bill or VA health care. We&apos;ll walk you
-      through the process step-by-step.
-    </p>
-    {/* eslint-disable-next-line jsx-a11y/no-redundant-roles */}
-    <ul
-      className="usa-grid usa-grid-full vads-u-margin-top--3 vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row"
-      role="list"
-    >
-      <li className="featured-content-list-item vads-u-background-color--primary-alt-lightest  vads-u-padding-y--1p5 vads-u-padding-x--1p5 vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0 vads-u-display--flex vads-u-flex-direction--column">
-        <h3 className="vads-u-font-size--base vads-u-margin--0">
-          File a VA disability claim
-        </h3>
-        <hr
-          aria-hidden="true"
-          role="presentation"
-          className="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary"
-        />
-        <p className="va-nav-linkslist-description">
-          Equal to VA Form 21-526EZ
-        </p>
-        <a
-          className="vads-u-display--block vads-u-padding-top--1 vads-u-text-decoration--none"
-          href="/disability/file-disability-claim-form-21-526ez/"
-          onClick={onFeaturedContentClick('File a VA disability claim')}
-        >
-          <span>
-            Apply online
-            <span className="vads-u-visibility--screen-reader">
-              about filing a VA disability claim
+export const FindVaForms = ({ showPdfWarningBanner }) => {
+  useEffect(
+    () => {
+      if (showPdfWarningBanner) {
+        const PDFAlertContent = `<va-alert>
+        <h2 slot="headline" className="vads-u-font-size--h3">
+          We’re updating our forms
+        </h2>
+        <p>After January 7, 2023, you won't be able to use VA forms that have a “last updated” date before March 2022. If you downloaded any of these older VA forms, you may need to download new copies in January.</p>
+        </va-alert>`;
+
+        document
+          .getElementsByClassName('va-introtext')[0]
+          .insertAdjacentHTML('beforebegin', `${PDFAlertContent}`);
+      }
+    },
+    [showPdfWarningBanner],
+  );
+
+  return (
+    <>
+      <SearchForm />
+      <SearchResults />
+      <h2>Frequently used VA forms</h2>
+      <p>
+        You can now do many form-based tasks online, like filing a disability
+        claim and applying for the GI Bill or VA health care. We’ll walk you
```

Comment:
On line 23, the old version of this text, the apostrophe in We'll was encoded, `We&apos;ll `. Does that need to be preserved?

Reply:
There was a lint rule advising to change it